magicctapipe/scripts/lst1_magic/lst1_magic_dl2_to_dl3.py

```diff
@@ -159,9 +206,8 @@ def create_event_list(event_table, effective_time, elapsed_time,
     time_start = Time(event_table['timestamp'][0], format='unix', scale='utc')
     time_end = Time(event_table['timestamp'][-1], format='unix', scale='utc')
 
-    delta_time = time_end.value - time_start.value
-
-    deadc = effective_time / elapsed_time
+    elapsed_time = time_end.value - time_start.value
```

Comment:
not sure over how big chunks of data this function can be run, but a safer way to compute the elapsed time is to take all the time difference, exclude big gaps (e.g. > 0.1 s), and sum them